notanorm/base.py

```diff
@@ -840,15 +840,24 @@ def insert(self, table: str, ins=None, **vals):
 
     def auto_quote(self, key: str):
```

Comment:
Running into some additional quoting problems when trying to use a `GROUP BY` aggregation inside a subquery -- the fields are copied verbatim, which (a) breaks quoting, and (b) isn't the behavior we want.

SQL comes out looking like `select "tbl.time","COUNT(*) as count" from ([big query here]) as subq_5881bda892d0cf0122ef50e62156779d`

Reply:
if the field is named "tbl.time" in the subquery and tbl.time in the outer query, then it should be fine... right?

Reply:
wait, are you selecting an agg from a subq or doing an agg in a subq and selecting from that?   from your result it looks like the former, but your words imply the latter